cognite/client/data_classes/datapoints_subscriptions.py

```diff
@@ -376,16 +363,16 @@ def _metadata(key: str) -> list[str]:
 
 
 class DatapointSubscriptionFilterProperties:
-    description = ["description"]
-    external_id = ["externalId"]
+    description: ClassVar = ["description"]
+    external_id: ClassVar = ["externalId"]
```

Comment:
```suggestion
    description: ClassVar = ["description"]
    external_id: ClassVar = ["externalId"]
```
```suggestion
    description: ClassVar[list[str]] = ["description"]
    external_id: ClassVar[list[str]] = ["externalId"]
```

Reply:
Changed in a separate PR: https://github.com/cognitedata/cognite-sdk-python/pull/1324